content/definition.md

```diff
@@ -1,15 +1,40 @@
+7. **It encourages fair compensation.** The software project’s maintainers may, at their discretion, request remuneration in the form of code contributions, financial consideration, or other forms of voluntary support from organizations that derive commercial value from the software.
 
-Maintaining this definition is the collaborative work of the [Ethical Source Working Group](/apply). Pull requests and issues to our [source code repository](https://github.com/ethicalSource/ethicalsource.dev) are encouraged.
+Ethical Open Source projects are encouraged to use the "Open|Ethical" badge in their project’s documentation: 
+
+![https://img.shields.io/badge/open-ethical-%234baaaa](https://img.shields.io/badge/open-ethical-%234baaaa) 
+
+[https://img.shields.io/badge/open-ethical-%234baaaa](https://img.shields.io/badge/open-ethical-%234baaaa)
+
+## Ethically Licensed Open Source
+
+Software is considered **Ethically Licensed Open Source**\* (ELOS) if it meets *any* of the following additional criteria:
+
+* It is distributed under the "Hippocratic License" (https://firstdonoharm.dev), or a similar license that prohibits the redistribution of the software in modified, derivative, or collected work form from being relicensed to any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in conflict with the rights enumerated in the United Nations Universal Declaration of Human Rights (https://www.un.org/en/universal-declaration-human-rights/).
+* It is distributed under the "Anti 996" license (https://996.icu/), or a similar license that prohibits the redistribution of the software in modified, derivative, or collected work form to any individual or organization that violates the International Labor Standards Conventions and/or Recommendations published by the International Labour Organization at https://www.ilo.org/global/standards/introduction-to-international-labour-standards/conventions-and-recommendations/lang--en/index.htm.
+* It is distributed under an "Atmosphere" license (https://www.open-austin.org/atmosphere-license/), or a similar license that prohibits the redistribution of the software in modified, derivative, or collected work form to any individual or organization that is materially and directly invested in operations or facilities that have significant negative impact on the global global climate response, as documented by the United Nations Climate Change Secretariat (https://unfccc.int/resource/climateaction2020/media/1308/unfccc_spm_2018.pdf).
+
+\* *The term "open source" in this context refers to the broad experience of public and collaborative software development in contribution to the collective software commons. ELOS software licenses may or may not conform to the Open Source Initiative’s criteria for open source software licenses (https://opensource.org/osd-annotated).*
```

Comment:
Do we think we'd want to include Markdown links around the citations? I like the inline URLs for transparency, but I find that the URL draws my eye because it's highlighted as a link, and then I scan backwards for context.

```suggestion
* It is distributed under the ["Hippocratic License" (https://firstdonoharm.dev)](https://firstdonoharm.dev), or a similar license that prohibits the redistribution of the software in modified, derivative, or collected work form from being relicensed to any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in conflict with the rights enumerated in the [United Nations Universal Declaration of Human Rights (https://www.un.org/en/universal-declaration-human-rights/)](https://www.un.org/en/universal-declaration-human-rights/).
* It is distributed under the ["Anti 996" license (https://996.icu/)](https://996.icu/), or a similar license that prohibits the redistribution of the software in modified, derivative, or collected work form to any individual or organization that violates the International Labor Standards Conventions and/or Recommendations published by the International Labour Organization at https://www.ilo.org/global/standards/introduction-to-international-labour-standards/conventions-and-recommendations/lang--en/index.htm.
* It is distributed under an ["Atmosphere" license (https://www.open-austin.org/atmosphere-license/)](https://www.open-austin.org/atmosphere-license/), or a similar license that prohibits the redistribution of the software in modified, derivative, or collected work form to any individual or organization that is materially and directly invested in operations or facilities that have significant negative impact on the global global climate response, as documented by the United Nations Climate Change Secretariat (https://unfccc.int/resource/climateaction2020/media/1308/unfccc_spm_2018.pdf).

\* *The term "open source" in this context refers to the broad experience of public and collaborative software development in contribution to the collective software commons. ELOS software licenses may or may not conform to the Open Source Initiative’s [criteria for open source software licenses (https://opensource.org/osd-annotated)](https://opensource.org/osd-annotated).*
```

Reply:
Great idea, thanks!